Delphyne Compile Error - Geometries

Came back to the desktop to find the compile failed at the last step:

```
In file included from /usr/include/c++/5/memory:81:0,
                 from /mnt/mervin/workspaces/delphyne/src/delphyne/src/backend/automotive_simulator.h:6,
                 from /mnt/mervin/workspaces/delphyne/src/delphyne/src/backend/automotive_simulator.cc:3:
/usr/include/c++/5/bits/unique_ptr.h: In instantiation of ‘typename std::_MakeUniq<_Tp>::__single_object std::make_unique(_Args&& ...) [with _Tp = drake::geometry::GeometryInstance; _Args = {const Eigen::Transform<double, 3, 1, 0>, std::unique_ptr<drake::geometry::Box, std::default_delete<drake::geometry::Box> >}; typename std::_MakeUniq<_Tp>::__single_object = std::unique_ptr<drake::geometry::GeometryInstance>]’:
/mnt/mervin/workspaces/delphyne/src/delphyne/src/backend/geometry_wiring.h:94:61:   required from ‘const drake::systems::InputPort<T>& delphyne::WirePriusGeometry(const string&, drake::Isometry3<T>&, drake::systems::DiagramBuilder<T>*, drake::geometry::SceneGraph<T>*, std::set<drake::geometry::Identifier<drake::geometry::GeometryTag> >*) [with T = double; typename std::enable_if<std::is_same<T, double>::value, int>::type <anonymous> = 0; std::__cxx11::string = std::__cxx11::basic_string<char>; drake::Isometry3<T> = Eigen::Transform<double, 3, 1>]’
/mnt/mervin/workspaces/delphyne/src/delphyne/src/backend/automotive_simulator.cc:160:24:   required from ‘delphyne::AgentBase<T>* delphyne::AutomotiveSimulator<T>::AddAgent(std::unique_ptr<delphyne::AgentBase<T> >) [with T = double]’
/mnt/mervin/workspaces/delphyne/src/delphyne/src/backend/automotive_simulator.cc:485:16:   required from here
/usr/include/c++/5/bits/unique_ptr.h:765:69: error: no matching function for call to ‘drake::geometry::GeometryInstance::GeometryInstance(const Eigen::Transform<double, 3, 1>, std::unique_ptr<drake::geometry::Box, std::default_delete<drake::geometry::Box> >)’
     { return unique_ptr<_Tp>(new _Tp(std::forward<_Args>(__args)...)); }
                                                                     ^
In file included from /mnt/mervin/workspaces/delphyne/src/delphyne/src/backend/automotive_simulator.cc:18:0:
/mnt/mervin/workspaces/delphyne/install/include/drake/geometry/geometry_instance.h:97:3: note: candidate: drake::geometry::GeometryInstance::GeometryInstance(drake::Isometry3<double>&, std::unique_ptr<drake::geometry::Shape>, const string&, const drake::geometry::VisualMaterial&)
   GeometryInstance(const Isometry3<double>& X_PG, std::unique_ptr<Shape> shape,
   ^
/mnt/mervin/workspaces/delphyne/install/include/drake/geometry/geometry_instance.h:97:3: note:   candidate expects 4 arguments, 2 provided
/mnt/mervin/workspaces/delphyne/install/include/drake/geometry/geometry_instance.h:88:3: note: candidate: drake::geometry::GeometryInstance::GeometryInstance(drake::Isometry3<double>&, std::unique_ptr<drake::geometry::Shape>, const string&)
   GeometryInstance(const Isometry3<double>& X_PG, std::unique_ptr<Shape> shape,
```

Probably due to a drake api change (we recently shifted the hash)? Haven't had time to dig further yet.

@basicNew can you reproduce the problem?

--- a/src/backend/geometry_wiring.h
+++ b/src/backend/geometry_wiring.h
@@ -93,7 +93,8 @@ const drake::systems::InputPort<T>& WirePriusGeometry(
   // car chassis frame as its origin.
   auto car_bounding_box = std::make_unique<GeometryInstance>(
       Isometry3<T>::Identity(),
-      std::make_unique<Box>(kPriusCarLength, kPriusCarWidth, kPriusCarHeight));
+      std::make_unique<Box>(kPriusCarLength, kPriusCarWidth, kPriusCarHeight),
+      "prius_bounding_box");
 
   geometry_ids->insert(scene_graph->RegisterGeometry(
       source_id, car_chassis_frame_id, std::move(car_bounding_box)));

